mlst/2.23.0-2024-01/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:jammy as app
+
+ARG MLST_VER="2.23.0"
+ARG ANY2FASTA_VER="0.4.2"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="mlst"
+LABEL software.version="${MLST_VER}"
+LABEL description="Scan contig files against PubMLST typing schemes"
+LABEL website="https://github.com/tseemann/mlst"
+LABEL license="https://github.com/tseemann/mlst/blob/master/LICENSE"
+LABEL maintainer="Inês Mendes"
+LABEL maintainer.email="ines.mendes@theiagen.com"
+
+# copy README for posterity
```

Comment:
Oooh, I like this! Never thought of baking the README.md into the image! 

`COPY` Adds an additional layer, but I don't think this is a big deal. Might be slightly better to move this bit of code to the end of the `app` layer, right before the `FROM app as test` line 71. This way the caching of the `apt-get` layers may be used between different docker images.